src/mame/epoch/scv.cpp

```diff
@@ -587,6 +601,9 @@ ROM_START(scv)
 
 	ROM_REGION(0x400, "charrom", 0)
 	ROM_LOAD("epochtv.chr.s02", 0, 0x400, BAD_DUMP CRC(db521533) SHA1(40b4e44838c35191f115437a14f200f052e71509))
+
+	ROM_REGION(0x400, "upd1771c", 0)
+	ROM_LOAD16_WORD("upd1771c-017", 0, 0x400, CRC(975bd68d) SHA1(e777217c622331677ac1d6520a741d48ad9133c0))
```

Comment:
I'm not sure what MAME's rules are for ROM filenames, but should it have an extension? We've been naming this ROM "upd1771c-017.s03"... just because ".s01" and ".s02" are listed above it.

Reply:
Normally the location on the pcb is put in the extension like u1, or u2, but I have not found any pictures of the pcb where such markings are visible for upd1771c chip.

Reply:
TIL something new, thanks!
Here's a pic of my NTSC board. What a funny coincidence 😂 
![image](https://github.com/user-attachments/assets/87309753-c281-4fa5-bb6f-e9c8c98dcaf6)